Civi/Api4/Service/Spec/SpecGatherer.php

```diff
@@ -122,6 +122,7 @@ private function addCustomFields($entity, RequestSpec $spec) {
 
     $query = CustomField::get(FALSE)
       ->setSelect(['custom_group_id.name', 'custom_group_id.title', '*'])
+      ->addWhere('is_active', '=', TRUE)
       ->addWhere('custom_group_id.is_multiple', '=', '0');
```

Comment:
@colemanw do we need to do something about the multi record custom fields as well?